pyEpiabm/pyEpiabm/toy_population_config.py

```diff
@@ -99,6 +100,7 @@ def no_households(self, population: Population):
             considered for grouping
         :type population: Population
         """
+        # Will probably delete this and replace with assertion error.
```

Comment:
Is this comment still relevant? Or a future plan?

Reply:
Future plan